src/Picker/ArticlePickerProvider.php

```diff
@@ -82,7 +82,7 @@ protected function getLinkClass()
     /**
      * {@inheritdoc}
      */
-    protected function getRouteParameters(PickerConfig $config)
+    protected function getRouteParameters(PickerConfig $config = null)
```

Comment:
Yes, but `null` is never passed as argument, is it? We could leave it as it was.